### DISCUSSION TITLE
Header Banner Animations

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/site#754] [General]
# Header Banner Animations

Apply AOS animations on header banner title and description.

https://hbstack.dev/examples/header-banner-aos/

**Comment by pleasantone** [comment]
The English version of this page seems to have a corrupt or incomplete example?

**Reply by razonyang** [reply]
Hi, would you mind explaining the _corrupt_ in detail? Animation not working or sth else?

**Reply by pleasantone** [reply]
Sorry, I meant the source code section. It’s blank followed by dashes …  followed by a partial? example of source.

I’m unable to attach a photo from my phone.

**Reply by razonyang** [reply]
I think that's a service/network issue that you can't access the emgithub service, you can use the content panel (_pencil_ icon -> _view the page source_) to view the source code on GitHub.

**Reply by razonyang** [reply]
Resolved, I replaced the external service with function that reading the page params directly, now the source code will show the important/required parameters only.
